src/utilities/isEven.test.js

```diff
@@ -0,0 +1,29 @@
+import { isEven } from './isEven';
+
+describe('The isEven function', () => {
+  describe('when provided with even number', () => {
```

Comment:
Try using the `a`, `an`, and `the` articles appropriately throughout all of your tests. This slight problem is appearing in other files as well.

![image](https://user-images.githubusercontent.com/15181215/222012822-dcec5a82-afb1-4146-967f-9d5ae57f0080.png)